google-cloud-tools-plugin/google-account/src/com/google/cloud/tools/intellij/analytics/GoogleUsageTracker.java

```diff
@@ -111,7 +108,8 @@
   public GoogleUsageTracker() {
     analyticsId = UsageTrackerManager.getInstance().getAnalyticsProperty();
 
-    AccountPluginInfoService pluginInfo = ServiceManager.getService(AccountPluginInfoService.class);
+    AccountPluginInfoService pluginInfo = ServiceManager
```

Comment:
nit: This line break feels weird; did the auto-formatter do this? I would expect `ServiceManager` to be on the next line with the method call.

Reply:
fixed.